packages/design-system/src/components/SearchBar/style.scss

```
@@ -50,6 +51,10 @@ $base-class: "search-bar";
       box-sizing: border-box;
       border: 1px solid #8db4ee;
     }
+
+    &--disable {
```

```suggestion
    &--hidden {
```
`disable` usually means something different